crates/client/eth/Cargo.toml

```diff
@@ -0,0 +1,70 @@
+[package]
+authors.workspace = true
+description = "This crate is responsible to handle l1 communication"
+edition = "2021"
```

Comment:
Please use
```rs
authors.workspace = true
edition.workspace = true
license.workspace = true
repository.workspace = true
version.workspace = true
homepage.workspace = true
```

Can you also fix that in the `Cargo.toml` of the sync crate? seems we missed it :)

Reply:
fixed